### DISCUSSION TITLE
How do i build the apk for android?

### DISCUSSION THREAD

**Discussion by hlarc** [FossifyOrg/General-Discussion#591] [Q&A]
There doesnt seem to be precompiled apks for Android in the releases page. And if i run the gradle Script it says the build was succesful but i cant find the compiled version anywhere. Maybe I did something wrong? But i couldnt find Documentation with Steps to Build it yourself.

**Comment by naveensingh** [comment]
Hello!

> if i run the gradle Script it says the build was succesful but i cant find the compiled version anywhere.

The exact location depends on the type of build flavor. Generated APKs will be located somewhere in `app/build/outputs/apk/...`.

Alternatively, you can also just download the last Simple Draw release from here: https://github.com/SimpleMobileTools/Simple-Draw/releases/tag/6.9.6

More info: https://github.com/FossifyOrg/Camera/discussions/17#discussioncomment-10146218
